src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -227,6 +229,22 @@ def has_connected_lms(course_run):
     return LMSHandler.select_lms(course_run.resource_link) is not None
 
 
+@register.filter()
+def is_joanie_product(course_run):
+    """
+    Check that the course run is managed by Joanie
+    then the resource type is equal to `products`.
+    """
+    handler = LMSHandler.select_lms(course_run.resource_link)
+
+    if not isinstance(handler, JoanieBackend):
+        return False
```

Comment:
Should we use the new class attribute?
```suggestion
    if not getattr(handler, "is_joanie", False):
        return False
```

Reply:
Yep. It improves cohesion.